tests/integrations/pymongo/test_pymongo.py

```diff
@@ -62,22 +62,23 @@ def test_transactions(sentry_init, capture_events, mongo_server, with_pii):
         assert span["data"][SPANDATA.SERVER_PORT] == mongo_server.port
         for field, value in common_tags.items():
             assert span["tags"][field] == value
+            assert span["data"][field] == value
 
     assert find["op"] == "db"
     assert insert_success["op"] == "db"
     assert insert_fail["op"] == "db"
 
-    assert find["tags"]["db.operation"] == "find"
-    assert insert_success["tags"]["db.operation"] == "insert"
-    assert insert_fail["tags"]["db.operation"] == "insert"
+    assert find["data"]["db.operation"] == "find"
+    assert insert_success["data"]["db.operation"] == "insert"
+    assert insert_fail["data"]["db.operation"] == "insert"
```

Comment:
Since we are going the backwards-compatible route, please keep the assertions that make sure we set the tags, as well